llama_stack/apis/inference/inference.py

```diff
@@ -60,6 +69,104 @@ class Int4QuantizationConfig(BaseModel):
 ]
 
 
+@json_schema_type
+class ImageContentItem(BaseModel):
```

Comment:
are these types common enough (even used in apis like finetuning) that they belong in common instead of having to import inference everywhere?

Reply:
yeah this is a good point. the type does show up in a few other places. I will move it to somewhere in apis/common/